common/corerequestcontext.go

```diff
@@ -114,6 +97,16 @@ func CoreRequestContextMiddleware(logger *logrus.Logger) func(next http.Handler)
 
 type coreRequestContextKey struct{}
 
+// Deprecated: github.com/anz-bank/pkg/log is replacing the logrus
+func GetLogEntryFromContext(ctx context.Context) *logrus.Entry {
```

Comment:
Please move these back to their previous position within the file. It make reviewing and conflicts easier to manage.